vision/src/autogluon/vision/configs/presets_configs.py

```diff
@@ -0,0 +1,184 @@
+"""Presets for vision predictors"""
+import functools
+from autogluon.core import Categorical, Real
+
+
+def unpack(preset_name):
+    if not preset_name in _PRESET_DICTS:
+        raise ValueError(f'Unknown preset_name: {preset_name}')
+    def _unpack_inner(f):
+        @functools.wraps(f)
+        def _call(*args, **kwargs):
+            gargs, gkwargs = set_presets(preset_name, *args, **kwargs)
+            return f(*gargs, **gkwargs)
+        return _call
+    return _unpack_inner
+
+
+# Dictionary of preset fit() parameter configurations for ImagePredictor.
+preset_image_predictor = dict(
+    # Best predictive accuracy with little consideration to inference time or model size. Achieve even better results by specifying a large time_limit value.
+    # Recommended for applications that benefit from the best possible model accuracy.
+    best_quality={
+        'hyperparameters': {
+            'model': Categorical('resnet50_v1b', 'resnet101_v1d', 'resnest200'),
+            'lr': Real(1e-2, 1e-6, log=True),
+            'batch_size': Categorical(8, 16, 32, 64, 128),
+            'epochs': 200
+            },
+        'hyperparameter_tune_kwargs': {
+            'num_trials': 1024,
+            'search_strategy': 'bayesopt',
+            'max_reward': 1.0
+        }
+    },
+
+    # Good predictive accuracy with fast inference.
+    # Recommended for applications that require reasonable inference speed and/or model size.
+    good_quality_fast_inference={
+        'hyperparameters': {
+            'model': Categorical('resnet50_v1b', 'resnet34_v1b'),
+            'lr': Real(1e-2, 1e-6, log=True),
+            'batch_size': Categorical(8, 16, 32, 64, 128),
+            'epochs': 150
+            },
+        'hyperparameter_tune_kwargs': {
+            'num_trials': 512,
+            'search_strategy': 'bayesopt',
+            'max_reward': 0.95
+        }
+    },
+
+    # Medium predictive accuracy with very fast inference and very fast training time. 
+    # This is the default preset in AutoGluon, but should generally only be used for quick prototyping.
+    medium_quality_faster_train={
+        'hyperparameters': {
+            'model': 'resnet50_v1b',
+            'lr': 0.01,
+            'batch_size': 64,
+            'epochs': 50
+            },
+        'hyperparameter_tune_kwargs': {
+            'num_trials': 8,
+            'search_strategy': 'random',
+            'max_reward': 0.85
```

Comment:
Why 0.85? This seems extremely low. I generally think we should avoid any specification of max_reward, since an acceptable accuracy differs drastically between problems.